hardware_device_context.go

```diff
@@ -0,0 +1,37 @@
+package astiav
+
+//#cgo pkg-config: libavutil libavcodec
+//#include <libavcodec/avcodec.h>
+//#include <libavutil/hwcontext.h>
+import "C"
+import (
+	"unsafe"
+)
+
+// https://github.com/FFmpeg/FFmpeg/blob/n5.0/libavutil/hwcontext.h#L141
```

Comment:
Could you replace the link with `https://github.com/FFmpeg/FFmpeg/blob/n5.0/libavutil/hwcontext.h#L61`?